docs/pages/server-access/guides/openssh.mdx

```diff
@@ -117,11 +148,44 @@ host.
 
 ### Issue a host certificate
 
-On your local machine, assign the IP address or fully qualified domain name of
-your Node to an environment variable.
+#### Find your node's UUID
+
+When you created a `node` resource earlier, the Teleport Auth Service generated a
+universal unique identifier (UUID) for that node. Teleport Proxy Services uses
+the UUID to differentiate nodes with the same hostname, so it be added to the host certificate.
+To find your node's UUID, first determine if its hostname is unique:
 
 ```code
-$ ADDR=203.0.113.0
+$ tctl get node/openssh-node --format text
+```
+
+If only one node is displayed and you have `jq` installed, you can run the
+following command to get your node's UUID:
+
+```code
+$ tctl get node/openssh-node --format=json | jq -r ".[0].metadata.name"
+```
+
+Otherwise, find your node's UUID in the `metadata.name` field of the YAML
+output of this command:
+
+```code
+$ tctl get node/openssh-node
+```
+
+#### Create the host certificate
+
+When creating host certificates, it is important to specify all the domain names
+and addresses that refer to your node. If you try to connect to a node with a 
+name or address that was not specified when creating it's host certificate,
+Teleport will reject the SSH connection.
+
+On your local machine, assign the IP address, fully qualified domain name of
+your node, and the node's UUID to an environment variable. If you won't be
+connecting to your node with its hostname, you can safely omit it.
+
+```code
+$ ADDR=1.2.3.4,openssh-node,a100fdd0-52db-4eca-a7ab-c3afa7a1564a
```

Comment:
Is finding/including the node's UUID required?

Reply:
I guess my question was more of "why" is it required? Is it required currently for OpenSSH nodes too? Just thinking that having to find out this ID might complicate automation for users.

Reply:
Unregistered OpenSSH nodes don't have UUIDs assigned to them, as they are not a registered Teleport resource, so that was not required.

Reply:
@capnspacehook When you're creating the OpenSSH server resource, you can assign the UUID yourself, correct? Think it would make sense to mention this in the docs? Then users wouldn't need to fetch the resource back after creating it at least.

Reply:
True, fixed